### PR TITLE
feat(Reports): change message domain non verified

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Remove message under of dropdown and put before reports boxes with other design
- Add test to check this state

image

### CODE REVIEW

File: src/components/Reports/ReportsFilters/ReportsFilters.js [hunk 1]
```diff
@@ -50,9 +49,7 @@ const ReportsFilters = ({
                             <FormattedDate value={domainSelected.verified_date} />
                           </span>
                         </>
-                      ) : (
-                        <FormattedMessageMarkdown id="reports_filters.domain_not_verified_MD" />
-                      )}
+                      ) : null}
```
Comment (cbernat) [inline src/components/Reports/ReportsFilters/ReportsFilters.js]: Are you sure a null value here is ok?

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsFilters/ReportsFilters.js]: because the message now show before the boxes

**Comment by cbernat** [comment]
🚀 I see this ok!

**Pull request merged**